src/main/java/frc/robot/classes/ChooChooMap.java

```diff
@@ -0,0 +1,478 @@
+package frc.robot.classes;
+
+import java.util.HashMap;
+
+public class ChooChooMap{
+    private HashMap<Double, Double> chooChooMap = new HashMap<Double,Double>();
+
+    public double toChooChooAngle(double armAngle){
+        return chooChooMap.get(truncate(armAngle, 1));
+    }
+
+    private double truncate(double a, int places){
+        return Math.round(a * Math.pow(10, places)) / Math.pow(10, places);
+    }
+
+    public ChooChooMap(){
```

Comment:
Include the code used to generate this list (somewhere outside the src directory so gradle doesn't find)
Add comments to what this class is (desmos links, how it was calculated, etc.